.github/workflows/copyrights.yml

```diff
@@ -0,0 +1,22 @@
+name: copyrights
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  copyrights:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: yegor256/copyrights-action@0.0.8
+        with:
+          license: LICENSE.txt
+          globs: >-
+            **/*.yml
+            **/*.yaml
+            **/*.hs
+            **/*.sh
+            **/*.phi
```

Comment:
Include  `.cf` files (we have [Syntax.cf](https://github.com/objectionary/eo-phi-normalizer/blob/c53ed6a49b5632c4eed7cc347d1fb3bfc1f7361e/eo-phi-normalizer/grammar/EO/Phi/Syntax.cf)).